package.json

```diff
@@ -96,9 +97,11 @@
     "opn-cli": "^3.1.0",
     "phantomjs-prebuilt": "^2.1.7",
     "pretty-bytes-cli": "^2.0.0",
+    "prompt": "^1.0.0",
     "replace-ext": "1.0.0",
     "rimraf": "^2.5.2",
     "semver": "^5.1.0",
+    "shelljs": "^0.7.8",
```

Comment:
Higher level API + cross platform implementation of common commands. Not fully exploited here but I think it's quite nice :)